.gitignore

```diff
@@ -6,3 +6,4 @@ repos/*
 *.gemspec
```

Comment:
I've just noticed there are a few Ruby entries in `.gitignore` that can be removed now :-)

ie: Pretty much all entries aside from `.DS_Store`, `target/` and `tmp/`

Reply:
❤️